markdown/events/2021/bytesize-6-dsl2-using-modules-pipelines.md

```diff
@@ -38,13 +38,13 @@ The first talk will describe how to use DSL2 modules when writing an nf-core pip
 
 **Note: This text has been edited to make it more suitable for reading.**
 
-### DSL2 - Using modules in a pipeline
+#### DSL2 - Using modules in a pipeline
 
 Thank you very much and welcome to the bytesize talk on how to use modules in a pipeline.
 
 I would just like to briefly recap what a module is. It is an atomic process that cannot be reduced any further and usually contains a single software tool like FastQC for example and it can be used within a pipeline and also shared between different pipelines. To make use of the sharing, there is an nf-core modules repository on GitHub where you can find many of these modules already.
 
-So to make use of this modules repository, there’s a new sub-command in nf-core/tools and for a brief recap, you can install it with ´pip install nf-core´ and ´conda install nf-core´ and if you then run ´nf-core modules´, you get a list of sub-commands that you can use to interact with this repository. 
+So to make use of this modules repository, there’s a new sub-command in nf-core/tools and for a brief recap, you can install it with ´pip install nf-core´ and ´conda install nf-core´ and if you then run ´nf-core modules´, you get a list of sub-commands that you can use to interact with this repository.
```

Comment:
```suggestion
So to make use of this modules repository, there’s a new sub-command in nf-core/tools and for a brief recap, you can install it with `pip install nf-core` and `conda install nf-core` and if you then run `nf-core modules`, you get a list of sub-commands that you can use to interact with this repository.
```

Reply:
might as well do this replacement everywhere rather than the odd single quotes here as then you code blocks which is basically what these are

Reply:
Thanks. Good to learn how to use the quote marks for code blocks. :-) Thought they were the other way around.

Reply:
Changed them throughout the text. Thanks :-)

Reply:
I always use this page when I forget various Markdown things :)

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code